tests/test_integrations.py

```diff
@@ -0,0 +1,16 @@
+from dotenv import load_dotenv
+from langfuse.integrations import openai
+
+load_dotenv()
+
+
+def test_openai_chat_completion():
+    completion = openai.ChatCompletion.create(
+        model="gpt-3.5-turbo", messages=[{"role": "user", "content": "1 + 1 = "}], temperature=0
+    )
+    assert len(completion.choices) != 0
+
+
+def test_openai_completion():
+    completion = openai.Completion.create(model="gpt-3.5-turbo-instruct", prompt="1 + 1 = ", temperature=0)
+    assert len(completion.choices) != 0
```

Comment:
can you expose the flush function and call it in the tests? Also, did you try running this locally?

Reply:
```
    api = FintoLangfuse(
        username=os.environ.get("LF_PK"), password=os.environ.get("LF_SK"), environment=os.environ.get("HOST")
    )
    api.generations.get()
```
could you also fetch all the generations and check that the one you generate is there? Maybe adding a UUID as a prompt helps to find it?

Reply:
I tested it locally and on UI it was showing properly @maxdeichmann

Reply:
> can you expose the flush function and call it in the tests? Also, did you try running this locally?

I'm actually doing it inside a method. Do I need to call it again in tests?

```  
def _log_result(self, result, call_details):
      generation = InitialGeneration(name="OpenAI-generation", **call_details)
      self.langfuse.generation(generation)
      self.langfuse.flush()
      return result
```

Reply:
> ```
>     api = FintoLangfuse(
>         username=os.environ.get("LF_PK"), password=os.environ.get("LF_SK"), environment=os.environ.get("HOST")
>     )
>     api.generations.get()
> ```
> 
> could you also fetch all the generations and check that the one you generate is there? Maybe adding a UUID as a prompt helps to find it?

It's throwing this `Pydantic` error. But I have made changes to include custom `trace_name` while making the call. Can check the last commit @maxdeichmann 
![image](https://github.com/langfuse/langfuse-python/assets/57898986/1bdad334-28da-4081-bea0-19f8916fe969)

Reply:
> I'm actually doing it inside a method. Do I need to call it again in tests?

ahh awesome, did not see that!

Reply:
I fixed it here: https://github.com/langfuse/langfuse/commit/528f9ee2bf01640408394df14aa37e3b79686440